RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -14,11 +14,15 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using Microsoft.CodeAnalysis;
 
 namespace SonarQube.Plugins.Roslyn
 {
     public class AnalyzerPluginGenerator
     {
+        // TODO: multiple language support
+        private static readonly string language = LanguageNames.CSharp;
```

Comment:
Constants should start with an upper-case letter. However, this constant isn't adding any value.